pixeldata/src/attribute.rs

```diff
@@ -507,4 +594,71 @@ mod tests {
             size
         );
     }
+
+    #[test]
+    fn test_get_rescale_intercept(){
+        let mut dcm = FileDicomObject::new_empty_with_meta(
+            FileMetaTableBuilder::new()
+                .transfer_syntax(uids::EXPLICIT_VR_LITTLE_ENDIAN)
+                .media_storage_sop_class_uid("1")
```

Comment:
I'd be inclined to specifying a real, standard SOP class UID, so as to open up the possibility of it being looked up during the decoding process.

```suggestion
                .media_storage_sop_class_uid(uuid::BREAST_TOMOSYNTHESIS_IMAGE_STORAGE)
```